src/lib.rs

```diff
@@ -216,11 +217,13 @@ pub use crate::Direction as EdgeDirection;
 
 /// Marker type for a directed graph.
 #[derive(Copy, Debug)]
+#[cfg_attr(feature = "serde-1", derive(serde::Serialize, serde::Deserialize))]
 pub enum Directed {}
```

Comment:
I think this serde derive can be removed. This type never has any values (and if it would have, it would be an error.)